src/dialect/sqlite.js

```diff
@@ -103,43 +101,37 @@ class Sqlite extends Dialect {
    * @return String       The SQL column string
    */
   _column(field) {
-    var name = field.name;
-    var use = field.use;
-    var type = field.type;
-    var length = field.length;
-    var precision = field.precision;
-    var serial = field.serial;
-    var nil = field.null;
-    var dft = field['default'];
+    let use = field.use;
+    const name = field.name;
+    const type = field.type;
+    const length = field.length;
+    const precision = field.precision;
+    const serial = field.serial;
+    const nil = field.null;
 
     if (type === 'float' && precision) {
       use = 'numeric';
     }
 
-    var column = this.name(name) + ' ' + this._formatColumn(use, length, precision);
-
-    var result = [column];
+    const column = this.name(name) + ' ' + this._formatColumn(use, length, precision);
+    const result = [column];
```

Comment:
I'd pull the next `result.push()` up

Reply:
the meta('column') one?
pull it up to what?

Reply:
oh. its unconditional, so just put it in the array.